splunk_connect_for_snmp/inventory/tasks.py

```diff
@@ -284,12 +284,20 @@ def create_profile(profile_name, frequency, varBinds, records):
 
 
 def create_query(conditions: typing.List[dict], address: str) -> dict:
-
     conditional_profiles_mapping = {
         "equals": "$eq",
         "gt": "$gt",
         "lt": "$lt",
         "in": "$in",
+        "regex": "$regex",
+    }
+
+    negative_profiles_mapping = {
+        "equals": "$ne",
+        "gt": "$lte",
```

Comment:
is it `$lte`? not `$lt`?

Reply:
Yes, I followed with the logic as if we used `not` operator on `gt` that would return something less or equal.

Reply:
ok, got it